recipes/libuvc/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "libuvc" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 42175a53c1c704365fdc782b44233925e40c9344fbb7f942181c1090f06e2873
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+    - pkgconfig
```

`pkgconfig` is the Python package, you probably only need the executable.
```suggestion
    - pkg-config
```